posts/qutip_10_years.html

```diff
@@ -0,0 +1,176 @@
+            background: #fff;
+            clear: left;
+            font: 14px "Lucida Console", Monaco, monospace;
+            width: 100%;
+        }
+
+        #mce-EMAIL {
+            padding: 11px;
+            margin: 10px;
+        }
+    </style>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-19932157-4"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+
+        function gtag() {
+            dataLayer.push(arguments);
+        }
+        gtag('js', new Date());
+
+        gtag('config', 'UA-19932157-4');
+    </script>
+</head>
+
+<body>
+    <div class="layout">
+        <main id="content" class="content">
+            <section class="heavy">
+                <div class="container">
+                    <div class="col-3">
+                        <div class="hero">
+                            <a href="/"><img src="/logos/logov3.svg" alt="Unitary Fund" /></a>
+                            <p class="tagline"><b>Because evolution is
+                                    <a href="https://en.wikipedia.org/wiki/Unitarity_(physics)"
+                                        target="_blank">unitary</a></b>.</p>
+                            <button id="mobile-menu-icon" class="hamburger hamburger--squeeze" type="button">
+                                <span class="hamburger-box">
+                                    <span class="hamburger-inner"></span>
+                                </span>
+                            </button>
+                        </div>
+                        <section class="light">
+                            <right>
+                                <h1 class="leading-arrow">QuTiP's 10-year anniversary</h1>
+                            </right>
+                            <br> 9 July, 2021<br> The QuTiP Admin Team<br>
+                            <br>
+
+
+
+
+<p>QuTiP v. 1.0 was released 10 years ago, on July 29th, 2011. It began as the project of two postdocs in Franco Nori&rsquo;s group, Robert Johansson and Paul Nation and has since been embraced by an ever larger community of users and contributors. QuTiP is the most used software package for the simulation of open quantum systems as it&rsquo;s used daily by researchers worldwide and students alike.&nbsp;</p>
+<p><strong>To mark the 10 year anniversary</strong>, the QuTiP admin team would like to organize a <strong>online party</strong> to celebrate the project and its community of users and contributors: <strong>You are all invited to an online meeting on July 29th, 2021 at 1pm GMT</strong> (<a href="http://discord.unitary.fund/"><strong>on Unitary Fund&rsquo;s Discord server</strong></a>, see the </strong><a href="https://unitary.fund/talks.html#calendar">Community Calendar to save the event in your agenda). We'll share recent updates on the project and everyone is welcome to add their own contribution.&nbsp;</p>. As a summary, some stats:
```

Comment:
Hi @nathanshammah, I'm not very familiar with HTML syntax but is it possible that the link for the community calendar is not properly closed?
```
see the </strong><a href="https://unitary.fund/talks.html#calendar">Community Calendar to save the event in your agenda)
```